Angular production build failure with more than one lazy loading module

[dashboard.zip](https://github.com/jhipster/generator-jhipster/files/2108535/dashboard.zip)
##### **Overview of the issue**

Created another module named dashboard (very similar to "admin module" - structure wise). When importing this module at app.module level it works fine and production packaging working as expected. 
But when using lazy loading in app-routing.module (exactly same as admin module). Dev (./mvnw and yarn start) is working fine and I am able to see everything is loaded as expected. 
But production build keeps on failing to generate www package in target folder.
( ./mvnw -Pprod clean package -DskipTests=true) 

##### **Motivation for or Use Case**

It is stopping to package the web application for production which I am deploying to google cloud using CI build

##### **Reproduce the error**
1) Build jhipster microservice gateway without any entity 
2) create module dashboard under app folder (equal level as admin, core etc).  I have attached the folder with code.
3) build process is successful but not generating www folder. please see log message.

##### **Related issues**

<!-- Has a similar issue been reported before? Please search both closed & open issues -->

##### **Suggest a Fix**

it has something to do with webpack production build when using lazy loading ( used --progress flag in production build but not really helpful). When packaging is successful I am seeing it is emmitting and then displaying webpack assets and chunks. but in this scenario is it failing to do anything and going directly to $ rimraf target//www/app/{src,target/}

**See logs of webpack build for production.**  

<details>
<summary>Webpack output</summary>
<pre>
[INFO] --- frontend-maven-plugin:1.6:yarn (webpack build prod) @ gateway ---
[INFO] yarn not inheriting proxy config from Maven
[INFO] Running 'yarn run webpack:prod' in /Users/capalytix/Documents/DHUB/gateway
[INFO] yarn run v1.6.0
[INFO] $ yarn run cleanup && yarn run webpack:prod:main && yarn run clean-www
[INFO] $ rimraf target/{aot,www}
[INFO] $ yarn run webpack --config webpack/webpack.prod.js --progress --profile
[INFO] $ node --max_old_space_size=4096 node_modules/webpack/bin/webpack.js --config webpack/webpack.prod.js --progress --profile
[ERROR]   0% compiling(node:39630) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead                                                                       21888ms building modules                                                                  
[ERROR] 74ms finish module graph                            
[ERROR] 8ms sealing                                
[ERROR] 0ms basic dependencies optimization 
[ERROR] 78ms dependencies optimization                      
[ERROR] 0ms advanced dependencies optimization 
[ERROR] 43ms after dependencies optimization
[ERROR] 0ms optimizing 
[ERROR] 0ms basic module optimization 
[ERROR] 0ms module optimization 
[ERROR] 58ms advanced module optimization                          
[ERROR] 1ms after module optimization 
[ERROR] 4ms basic chunk optimization                             
[ERROR] 0ms chunk optimization 
[ERROR] 38ms advanced chunk optimization                        
[ERROR] 1ms after chunk optimization 
[ERROR] 1ms module and chunk tree optimization                   
[ERROR] 6266ms building modules                                                                  
[ERROR] 4ms module and chunk tree optimization                   
[ERROR] 1ms after module and chunk tree optimization 
[ERROR] 0ms basic chunk modules optimization 
[ERROR] 410ms chunk modules optimization                         
[ERROR] 1ms advanced chunk modules optimization 
[ERROR] 0ms after chunk modules optimization 
[ERROR] 0ms module reviving                 
[ERROR] 7ms module order optimization                       
[ERROR] 0ms advanced module order optimization 
[ERROR] 1ms before module ids 
[ERROR] 0ms module ids 
[ERROR] 11ms module id optimization
[ERROR] 0ms chunk reviving                 
[ERROR] 1ms chunk order optimization                       
[ERROR] 0ms before chunk ids 
[ERROR] 1ms chunk id optimization                          
[ERROR] 8ms after chunk id optimization 
[ERROR] 1ms record modules                 
[ERROR] 0ms record chunks                 
[ERROR] 378ms hashing
[ERROR] 0ms after hashing 
[ERROR] 0ms record hash 
[ERROR] 1ms module assets processing 
[ERROR] 976ms chunk assets processing
[ERROR] 4ms additional chunk assets processing 
[ERROR] 0ms recording 
[ERROR] 1301ms additional asset processing                
[ERROR] 33874ms chunk asset optimization            
[ERROR] 0ms after chunk asset optimization 
[ERROR] 1ms asset optimization 
[ERROR] 0ms after asset optimization 
[ERROR] 1ms after seal 
[INFO] $ rimraf target//www/app/{src,target/}
[INFO] Done in 74.39s.
[INFO] 
[INFO] --- maven-compiler-plugin:3.7.0:compile (default-compile) @ gateway ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 91 source files to /Users/capalytix/Documents/DHUB/gateway/target/classes
[INFO] 
[INFO] --- maven-resources-plugin:3.1.0:testResources (default-testResources) @ gateway ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 5 resources
[INFO] Copying 2 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.7.0:testCompile (default-testCompile) @ gateway ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 23 source files to /Users/capalytix/Documents/DHUB/gateway/target/test-classes
[INFO] 
[INFO] --- maven-surefire-plugin:2.21.0:test (default-test) @ gateway ---
[INFO] Tests are skipped.
[INFO] 
[INFO] --- maven-surefire-plugin:2.21.0:test (without-cucumber) @ gateway ---
[INFO] Tests are skipped.
[INFO] 
[INFO] --- jacoco-maven-plugin:0.8.1:report (post-unit-test) @ gateway ---
[INFO] Skipping JaCoCo execution due to missing execution data file.
[INFO] 
[INFO] --- frontend-maven-plugin:1.6:yarn (webpack build test) @ gateway ---
[INFO] Skipping execution.
[INFO] 
[INFO] --- maven-war-plugin:3.2.0:war (default-war) @ gateway ---
[INFO] Packaging webapp
[INFO] Assembling webapp [gateway] in [/Users/capalytix/Documents/DHUB/gateway/target/gateway-0.0.1-SNAPSHOT]
[INFO] Processing war project
[INFO] Copying webapp webResources [/Users/capalytix/Documents/DHUB/gateway/src/main/webapp] to [/Users/capalytix/Documents/DHUB/gateway/target/gateway-0.0.1-SNAPSHOT]
[INFO] Webapp assembled in [674 msecs]
[INFO] Building war: /Users/capalytix/Documents/DHUB/gateway/target/gateway-0.0.1-SNAPSHOT.war
[INFO] 
[INFO] --- spring-boot-maven-plugin:2.0.2.RELEASE:repackage (default) @ gateway ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------

**This is log of successful build without using lazy loading**

[INFO] --- frontend-maven-plugin:1.6:yarn (webpack build prod) @ gateway ---
[INFO] yarn not inheriting proxy config from Maven
[INFO] Running 'yarn run webpack:prod' in /Users/capalytix/Documents/DHUB/gateway
[INFO] yarn run v1.6.0
[INFO] $ yarn run cleanup && yarn run webpack:prod:main && yarn run clean-www
[INFO] $ rimraf target/{aot,www}
[INFO] $ yarn run webpack --config webpack/webpack.prod.js --progress --profile
[INFO] $ node --max_old_space_size=4096 node_modules/webpack/bin/webpack.js --config webpack/webpack.prod.js --progress --profile
[ERROR]   0% compiling(node:37671) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead                                                                       24488ms building modules                                                                  
[ERROR] 59ms finish module graph                            
[ERROR] 3ms sealing                                
[ERROR] 0ms basic dependencies optimization 
[ERROR] 102ms dependencies optimization                     
[ERROR] 0ms advanced dependencies optimization 
[ERROR] 63ms after dependencies optimization
[ERROR] 0ms optimizing 
[ERROR] 0ms basic module optimization 
[ERROR] 1ms module optimization 
[ERROR] 90ms advanced module optimization                          
[ERROR] 0ms after module optimization 
[ERROR] 4ms basic chunk optimization                             
[ERROR] 0ms chunk optimization 
[ERROR] 55ms advanced chunk optimization                        
[ERROR] 1ms after chunk optimization 
[ERROR] 2ms module and chunk tree optimization                   
[ERROR] 6668ms building modules                                                                  
[ERROR] 3ms module and chunk tree optimization                   
[ERROR] 1ms after module and chunk tree optimization 
[ERROR] 0ms basic chunk modules optimization 
[ERROR] 409ms chunk modules optimization                         
[ERROR] 0ms advanced chunk modules optimization 
[ERROR] 0ms after chunk modules optimization 
[ERROR] 1ms module reviving                 
[ERROR] 8ms module order optimization                       
[ERROR] 0ms advanced module order optimization 
[ERROR] 0ms before module ids 
[ERROR] 0ms module ids 
[ERROR] 19ms module id optimization
[ERROR] 0ms chunk reviving                 
[ERROR] 0ms chunk order optimization                       
[ERROR] 1ms before chunk ids 
[ERROR] 1ms chunk id optimization                          
[ERROR] 4ms after chunk id optimization 
[ERROR] 1ms record modules                 
[ERROR] 1ms record chunks                 
[ERROR] 376ms hashing
[ERROR] 0ms after hashing 
[ERROR] 0ms record hash 
[ERROR] 0ms module assets processing 
[ERROR] 1040ms chunk assets processing
[ERROR] 4ms additional chunk assets processing 
[ERROR] 0ms recording 
[ERROR] 295ms additional asset processing                 
[ERROR] 35454ms chunk asset optimization            
[ERROR] 0ms after chunk asset optimization 
[ERROR] 0ms asset optimization 
[ERROR] 1ms after asset optimization 
[ERROR] 1ms after seal 
[ERROR] 1424ms emitting                    
[INFO] Hash: 923982f53a4e78d71b3d
[INFO] Version: webpack 4.8.0
[INFO] Time: 75904ms
[INFO] Built at: 2018-06-16 20:37:33
[INFO]                                                 Asset       Size  Chunks                    Chunk Names
[INFO]                   swagger-ui/dist/lib/backbone-min.js   18.9 KiB          [emitted]         
[INFO]          content/510b7e991d47e560c0eb28ba3cbec928.svg   1.62 KiB          [emitted]         
[INFO]          content/9e21a58e0422c6f564eabf24df919054.jpg   8.99 KiB          [emitted]         
[INFO]          content/e9451a87f02911d2d4d0a5734389f9e2.jpg   8.99 KiB          [emitted]         
[INFO]          content/196d1d90c3e23ed8cdddbfa99be791d1.jpg   8.99 KiB          [emitted]         
[INFO]          content/9505fa5d074901758d26bf7dda01a676.jpg   8.99 KiB          [emitted]         
[INFO]          content/a0b2c65793eee92034ce6b0864343acb.jpg   10.1 KiB          [emitted]         
[INFO]          content/e8a115c341dd6083c4b0cce37854bf84.png    275 KiB          [emitted]  [big]  
[INFO]         content/ee0d38962fca58ff334f2179edf9d940.jpeg    117 KiB          [emitted]         
[INFO]                                       manifest.webapp  743 bytes          [emitted]         
[INFO]                   app/0.923982f53a4e78d71b3d.chunk.js   1.08 MiB       0  [emitted]  [big]  vendors
[INFO]                   app/1.923982f53a4e78d71b3d.chunk.js    163 KiB       1  [emitted]         
[INFO]               app/main.923982f53a4e78d71b3d.bundle.js    188 KiB       2  [emitted]         main
[INFO]             app/global.923982f53a4e78d71b3d.bundle.js   1.07 KiB       3  [emitted]         global
[INFO]          app/polyfills.923982f53a4e78d71b3d.bundle.js   1.28 KiB       4  [emitted]         polyfills
[INFO]          content/global-sass.923982f53a4e78d71b3d.css    477 KiB       3  [emitted]  [big]  global
[INFO]            content/main-sass.923982f53a4e78d71b3d.css    0 bytes       2  [emitted]         main
[INFO]                 content/main.923982f53a4e78d71b3d.css    0 bytes       2  [emitted]         main
[INFO]                                            robots.txt  239 bytes          [emitted]         
[INFO]                     swagger-ui/dist/swagger-ui.min.js    442 KiB          [emitted]  [big]  
[INFO]                                           favicon.ico    5.3 KiB          [emitted]         
[INFO]                    swagger-ui/dist/css/typography.css    0 bytes          [emitted]         
[INFO]                         swagger-ui/dist/css/reset.css  773 bytes          [emitted]         
[INFO]                         swagger-ui/dist/css/style.css   3.41 KiB          [emitted]         
[INFO]                        swagger-ui/dist/css/screen.css   42.6 KiB          [emitted]         
[INFO]          content/98c56720696c2f652cdff11ba3aad674.jpg   8.99 KiB          [emitted]         
[INFO]                         swagger-ui/dist/css/print.css   40.7 KiB          [emitted]         
[INFO]                       swagger-ui/dist/lib/es5-shim.js   22.2 KiB          [emitted]         
[INFO]  swagger-ui/dist/lib/highlight.9.1.0.pack_extended.js  310 bytes          [emitted]         
[INFO]           swagger-ui/dist/lib/highlight.9.1.0.pack.js   10.7 KiB          [emitted]         
[INFO]               swagger-ui/dist/lib/handlebars-4.0.5.js   69.8 KiB          [emitted]         
[INFO]             swagger-ui/dist/lib/jquery.slideto.min.js  365 bytes          [emitted]         
[INFO]              swagger-ui/dist/lib/jquery.ba-bbq.min.js   3.44 KiB          [emitted]         
[INFO]              swagger-ui/dist/lib/jquery.wiggle.min.js  536 bytes          [emitted]         
[INFO]               swagger-ui/dist/lib/jquery-1.8.0.min.js   89.9 KiB          [emitted]         
[INFO]                    swagger-ui/dist/lib/js-yaml.min.js   42.5 KiB          [emitted]         
[INFO]        swagger-ui/dist/lib/object-assign-pollyfill.js  349 bytes          [emitted]         
[INFO]                         swagger-ui/dist/lib/marked.js   15.4 KiB          [emitted]         
[INFO]                 swagger-ui/dist/lib/jsoneditor.min.js    127 KiB          [emitted]         
[INFO]                     swagger-ui/dist/lib/lodash.min.js   50.7 KiB          [emitted]         
[INFO]                  swagger-ui/dist/lib/swagger-oauth.js   6.88 KiB          [emitted]         
[INFO]              swagger-ui/dist/lib/sanitize-html.min.js    127 KiB          [emitted]         
[INFO]                                 swagger-ui/index.html   7.59 KiB          [emitted]         
[INFO]                   swagger-ui/dist/images/throbber.gif   9.04 KiB          [emitted]         
[INFO]                                        ./i18n/en.json   11.8 KiB          [emitted]         
[INFO]                                        ./i18n/fr.json   12.8 KiB          [emitted]         
[INFO]                                        ./i18n/de.json   12.4 KiB          [emitted]         
[INFO]                                            index.html   1.41 KiB          [emitted]         
[INFO] precache-manifest.4138c59092c5d7bc8b796085e1318c48.js    4.3 KiB          [emitted]         
[INFO]                                     service-worker.js  955 bytes          [emitted]         
[INFO] Entrypoint polyfills [big] = app/0.923982f53a4e78d71b3d.chunk.js app/polyfills.923982f53a4e78d71b3d.bundle.js
[INFO] Entrypoint global [big] = app/0.923982f53a4e78d71b3d.chunk.js app/global.923982f53a4e78d71b3d.bundle.js content/global-sass.923982f53a4e78d71b3d.css
[INFO] Entrypoint main [big] = app/0.923982f53a4e78d71b3d.chunk.js app/main.923982f53a4e78d71b3d.bundle.js content/main-sass.923982f53a4e78d71b3d.css content/main.923982f53a4e78d71b3d.css
[INFO]  [10] ./node_modules/@angular/platform-browser/fesm5/platform-browser.js + 1 modules 109 KiB {0} [built]
[INFO]       |    2 modules
[INFO]  [53] (webpack)/buildin/global.js 489 bytes {0} [built]
[INFO]       [622] 18437ms -> [467] 586ms -> factory:782ms building:158ms = 19963ms
[INFO]  [56] ./node_modules/@ng-bootstrap/ng-bootstrap/datepicker/datepicker.module.js + 4 modules 40.1 KiB {0} [built]
[INFO]       |    5 modules
[INFO]  [59] ./node_modules/@ngx-translate/core/index.js + 1 modules 10.1 KiB {0} [built]
[INFO]       |    2 modules
[INFO]  [69] ./node_modules/@ng-bootstrap/ng-bootstrap/accordion/accordion.module.js + 1 modules 8.5 KiB {0} [built]
[INFO]       |    2 modules
[INFO]  [70] ./node_modules/@ng-bootstrap/ng-bootstrap/tabset/tabset.module.js + 1 modules 8.3 KiB {0} [built]
[INFO]       |    2 modules
[INFO]  [86] ./node_modules/@ng-bootstrap/ng-bootstrap/carousel/carousel.module.js + 1 modules 9.79 KiB {0} [built]
[INFO]       |    2 modules
[INFO]  [87] ./node_modules/@ng-bootstrap/ng-bootstrap/typeahead/typeahead.module.js + 1 modules 14.4 KiB {0} [built]
[INFO]       |    2 modules
[INFO]  [96] ./node_modules/@ng-bootstrap/ng-bootstrap/dropdown/dropdown.module.js + 1 modules 10.7 KiB {0} [built]
[INFO]       |    2 modules
[INFO]  [97] ./node_modules/@ng-bootstrap/ng-bootstrap/rating/rating.module.js + 1 modules 7.69 KiB {0} [built]
[INFO]       |    2 modules
[INFO]  [98] ./node_modules/@ng-bootstrap/ng-bootstrap/timepicker/timepicker.module.js + 2 modules 12.8 KiB {0} [built]
[INFO]       |    3 modules
[INFO] [432] ./src/main/webapp/app/app.main.ts + 58 modules 276 KiB {2} [built]
[INFO]       | ./src/main/webapp/app/account/account.module.ts 172 bytes [built]
[INFO]       |     [] 18441ms -> [] 667ms -> factory:4137ms building:3ms dependencies:3897ms = 27145ms
[INFO]       | ./src/main/webapp/app/app.main.ts 532 bytes [built]
[INFO]       |     factory:5290ms building:13151ms = 18441ms
[INFO]       | ./src/main/webapp/app/app.module.ngfactory.js 23.5 KiB [built]
[INFO]       |     [] 18441ms -> factory:483ms building:184ms dependencies:4089ms = 23197ms
[INFO]       | ./src/main/webapp/app/layouts/main/main.component.ts 272 bytes [built]
[INFO]       |     [] 18441ms -> [] 667ms -> factory:4137ms building:3ms dependencies:3897ms = 27145ms
[INFO]       | ./src/main/webapp/app/account/password-reset/finish/password-reset-finish.component.ts 2 KiB [built]
[INFO]       |     [] 18441ms -> [] 667ms -> [] 4140ms -> factory:3751ms building:3ms dependencies:1ms = 27003ms
[INFO]       | ./src/main/webapp/app/app.module.ts 178 bytes [built]
[INFO]       |     [] 18441ms -> [] 667ms -> factory:4137ms building:3ms dependencies:3897ms = 27145ms
[INFO]       | ./src/main/webapp/app/entities/entity.module.ts 263 bytes [built]
[INFO]       |     [] 18441ms -> [] 667ms -> factory:4137ms building:3ms dependencies:3897ms = 27145ms
[INFO]       | ./src/main/webapp/app/blocks/config/prod.config.ts 265 bytes [built]
[INFO]       |     [] 18441ms -> factory:483ms building:184ms dependencies:4089ms = 23197ms
[INFO]       | ./src/main/webapp/app/public/public.module.ts 168 bytes [built]
[INFO]       |     [] 18441ms -> [] 667ms -> factory:4137ms building:3ms dependencies:3897ms = 27145ms
[INFO]       | ./src/main/webapp/app/public/home/home.module.ts 160 bytes [built]
[INFO]       |     [] 18441ms -> [] 667ms -> factory:4137ms building:3ms dependencies:3897ms = 27145ms
[INFO]       | ./src/main/webapp/app/layouts/frame/public-layout.component.ts 1.7 KiB [built]
[INFO]       |     [] 18441ms -> [] 667ms -> [] 4140ms -> factory:4019ms building:2ms dependencies:36ms = 27305ms
[INFO]       | ./src/main/webapp/app/layouts/frame/public-layout.component.ngfactory.js 2.8 KiB [built]
[INFO]       |     [] 18441ms -> [] 667ms -> factory:4137ms building:3ms dependencies:3897ms = 27145ms
[INFO]       | ./src/main/webapp/app/dashboard/dashboard.module.ts 152 bytes [built]
[INFO]       |     [] 18441ms -> [] 667ms -> factory:4137ms building:3ms dependencies:3897ms = 27145ms
[INFO]       | ./src/main/webapp/app/public/home/home.component.ts 1.22 KiB [built]
[INFO]       |     [] 18441ms -> [] 667ms -> [] 4140ms -> factory:3889ms building:1ms dependencies:1ms = 27139ms
[INFO]       | ./src/main/webapp/app/public/home/home.component.ngfactory.js 33.7 KiB [built]
[INFO]       |     [] 18441ms -> [] 667ms -> factory:4137ms building:3ms dependencies:3897ms = 27145ms
[INFO]       |     + 44 hidden modules
[INFO] [466] ./src/main/webapp/manifest.webapp 61 bytes {4} [built]
[INFO]       [622] 18437ms -> factory:469ms building:117ms dependencies:3359ms = 22382ms
[INFO] [622] ./src/main/webapp/app/polyfills.ts 2.42 KiB {4} [built]
[INFO]       factory:5289ms building:13148ms = 18437ms
[INFO] [626] ./node_modules/css-loader!./node_modules/postcss-loader/lib!./node_modules/sass-loader/lib/loader.js!./src/main/webapp/content/scss/global.scss 477 KiB [built]
[INFO]       [627] -> factory:760ms building:24434ms = 25194ms
[INFO]     + 614 hidden modules
[INFO] 
[INFO] WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
[INFO] This can impact web performance.
[INFO] Assets: 
[INFO]   content/e8a115c341dd6083c4b0cce37854bf84.png (275 KiB)
[INFO]   app/0.923982f53a4e78d71b3d.chunk.js (1.08 MiB)
[INFO]   content/global-sass.923982f53a4e78d71b3d.css (477 KiB)
[INFO]   swagger-ui/dist/swagger-ui.min.js (442 KiB)
[INFO] 
[INFO] WARNING in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
[INFO] Entrypoints:
[INFO]   polyfills (1.09 MiB)
[INFO]       app/0.923982f53a4e78d71b3d.chunk.js
[INFO]       app/polyfills.923982f53a4e78d71b3d.bundle.js
[INFO]   global (1.55 MiB)
[INFO]       app/0.923982f53a4e78d71b3d.chunk.js
[INFO]       app/global.923982f53a4e78d71b3d.bundle.js
[INFO]       content/global-sass.923982f53a4e78d71b3d.css
[INFO]   main (1.27 MiB)
[INFO]       app/0.923982f53a4e78d71b3d.chunk.js
[INFO]       app/main.923982f53a4e78d71b3d.bundle.js
[INFO]       content/main-sass.923982f53a4e78d71b3d.css
[INFO]       content/main.923982f53a4e78d71b3d.css
[INFO] 
[INFO] $ rimraf target//www/app/{src,target/}
[INFO] Done in 80.51s.
[INFO] 
[INFO] --- maven-compiler-plugin:3.7.0:compile (default-compile) @ gateway ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 91 source files to /Users/capalytix/Documents/DHUB/gateway/target/classes
[INFO] 
[INFO] --- maven-resources-plugin:3.1.0:testResources (default-testResources) @ gateway ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 5 resources
[INFO] Copying 2 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.7.0:testCompile (default-testCompile) @ gateway ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 23 source files to /Users/capalytix/Documents/DHUB/gateway/target/test-classes
[INFO] 
[INFO] --- maven-surefire-plugin:2.21.0:test (default-test) @ gateway ---
[INFO] Tests are skipped.
[INFO] 
[INFO] --- maven-surefire-plugin:2.21.0:test (without-cucumber) @ gateway ---
[INFO] Tests are skipped.
[INFO] 
[INFO] --- jacoco-maven-plugin:0.8.1:report (post-unit-test) @ gateway ---
[INFO] Skipping JaCoCo execution due to missing execution data file.
[INFO] 
[INFO] --- frontend-maven-plugin:1.6:yarn (webpack build test) @ gateway ---
[INFO] Skipping execution.
[INFO] 
[INFO] --- maven-war-plugin:3.2.0:war (default-war) @ gateway ---
[INFO] Packaging webapp
[INFO] Assembling webapp [gateway] in [/Users/capalytix/Documents/DHUB/gateway/target/gateway-0.0.1-SNAPSHOT]
[INFO] Processing war project
[INFO] Copying webapp webResources [/Users/capalytix/Documents/DHUB/gateway/src/main/webapp] to [/Users/capalytix/Documents/DHUB/gateway/target/gateway-0.0.1-SNAPSHOT]
[INFO] Copying webapp resources [/Users/capalytix/Documents/DHUB/gateway/target/www]
[INFO] Webapp assembled in [697 msecs]
[INFO] Building war: /Users/capalytix/Documents/DHUB/gateway/target/gateway-0.0.1-SNAPSHOT.war
[INFO] 
[INFO] --- spring-boot-maven-plugin:2.0.2.RELEASE:repackage (default) @ gateway ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------

</pre>
</details>


##### **JHipster Version(s)**

5.0.0.beta2

##### **JHipster configuration**
Using JHipster version installed locally in current project's node_modules
Executing jhipster:info
Options: 
Welcome to the JHipster Information Sub-Generator

##### **JHipster Version(s)**

```
gateway@0.0.0 /Users/capalytix/Documents/DHUB/gateway
└── generator-jhipster@5.0.0-beta.2 

```


##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**


<details>
<summary>.yo-rc.json file</summary>
<pre>
{
  "generator-jhipster": {
    "promptValues": {
      "packageName": "com.derivativehub.gateway",
      "nativeLanguage": "en"
    },
    "jhipsterVersion": "5.0.0-beta.2",
    "applicationType": "gateway",
    "baseName": "gateway",
    "packageName": "com.derivativehub.gateway",
    "packageFolder": "com/derivativehub/gateway",
    "serverPort": "8080",
    "authenticationType": "jwt",
    "cacheProvider": "hazelcast",
    "enableHibernateCache": true,
    "websocket": "spring-websocket",
    "databaseType": "sql",
    "devDatabaseType": "postgresql",
    "prodDatabaseType": "postgresql",
    "searchEngine": "elasticsearch",
    "messageBroker": false,
    "serviceDiscoveryType": "eureka",
    "buildTool": "maven",
    "enableSwaggerCodegen": false,
    "jwtSecretKey": "replaced-by-jhipster-info",
    "clientFramework": "angularX",
    "useSass": true,
    "clientPackageManager": "yarn",
    "testFrameworks": [
      "gatling",
      "cucumber",
      "protractor"
    ],
    "jhiPrefix": "jhi",
    "enableTranslation": true,
    "nativeLanguage": "en",
    "languages": [
      "en",
      "fr",
      "de"
    ]
  }
}
</pre>
</details>


##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

<details>
<summary>JDL entity definitions</summary>

<pre>

</pre>
</details>


##### **Environment and Tools**

java version "1.8.0_65"
Java(TM) SE Runtime Environment (build 1.8.0_65-b17)
Java HotSpot(TM) 64-Bit Server VM (build 25.65-b01, mixed mode)

git version 2.10.1 (Apple Git-78)

node: v8.9.4

npm: 6.1.0

yeoman: 2.0.2

yarn: 1.7.0

Docker version 18.03.0-ce-rc4, build fbedb97

docker-compose version 1.20.0-rc2, build 8c4af54

Congratulations, JHipster execution is complete!


##### **Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

no entity creation required to replicate this

##### **Browsers and Operating System**

Using Mac OSX El Capitan with Chrome Version 67.0.3396.87 (Official Build) (64-bit)


- [ yes] Checking this box is mandatory (this is just to show you read everything)

--- a/generators/client/templates/angular/webpack/webpack.common.js.ejs
+++ b/generators/client/templates/angular/webpack/webpack.common.js.ejs
@@ -105,7 +105,8 @@ module.exports = (options) => ({
         <%_ } _%>
         new HtmlWebpackPlugin({
             template: './<%= MAIN_SRC_DIR %>index.html',
-            chunksSortMode: 'dependency',
+            chunks: ['vendors', 'polyfills', 'global', 'main'],
+            chunksSortMode: 'manual',
             inject: 'body'
         })
     ]

